google/cloud/forseti/common/gcp_api/compute.py

```diff
@@ -304,10 +309,10 @@ def __init__(self, **kwargs):
         super(_ComputeBackendServicesRepository, self).__init__(
             component='backendServices', **kwargs)
 
-class _ComputeDisksRepository(
-        repository_mixins.AggregatedListQueryMixin,
-        repository_mixins.ListQueryMixin,
-        _base_repository.GCPRepository):
+
+class _ComputeDisksRepository(repository_mixins.AggregatedListQueryMixin,
```

Comment:
Why is this indentation preferred?  From what I can see, the original style is allowed, and should be preferred to make room for long args.

Reply:
https://www.python.org/dev/peps/pep-0008/#indentation

If we do need to accommodate longer arg lists then you can break re-do it but still adhering to the guidelines above.

Reply:
Yes, I did reference that.  But the guideline does allow the original indentation that was here.  So, why is this being changed?  And is it enforceable?  My issue is if we are expecting people to follow this, and make sure it's justified.

```
Yes:

# Aligned with opening delimiter.
foo = long_function_name(var_one, var_two,
                                           var_three, var_four)

# More indentation included to distinguish this from the rest.
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    print(var_one)

# Hanging indents should add a level.
foo = long_function_name(
    var_one, var_two,
    var_three, var_four)
```

Reply:
Yes, I hope to integrate pep8 checking (pip install pycodestyle). But we have a couple other issues it's complaining about that aren't fixable and it doesn't support disabling on a line level.